lib/main.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
```

Comment:
You don't need these include statements here if you put them in the header file.